src/Quiz.js

```diff
@@ -23,7 +44,7 @@ class Quiz extends Component {
                     submittedAnswers={this.props.submittedAnswers}
                     onAnswerClick={this.props.onAnswerClick}
                 />
-                {quizCompleted && <ContinueButton onClick={this.props.onContinueClick} />}
+                {questionCompleted && button}
             </div>
         )
     }
```

Comment:
I wouldn't say this is objectively better, but an alternate way to write this would be to put the entire "template" in the return statement:

```jsx
return (
    <div className="quiz">
       <IntervalExercise
           interval={this.props.interval}
           possibleAnswers={this.props.possibleAnswers}
           submittedAnswers={this.props.submittedAnswers}
           onAnswerClick={this.props.onAnswerClick}
       />
       {questionCompleted &&  (
           isLastExercise ? (
               <Button
                   text="Finish"
                   onClick={this.props.onFinishClick}
                   modifierClassName="finish-button"
               />
           ) : (
               <Button
                   text="Continue"
                   onClick={this.props.onContinueClick}
                   modifierClassName="continue-button"
               />
           )
       )}
    </div>
)
```

Totally up to you which way is more stylistically to your liking.